src/CommunityToolkit.Maui.Core/Views/Toast/PlatformToast.macios.cs

```diff
@@ -38,7 +38,8 @@ public class PlatformToast : Alert, IDisposable
 
 		messageLabel = new PaddedLabel(padding, padding, padding, padding)
 		{
-			Lines = new IntPtr(0)
+			Lines = new IntPtr(0),
+			AdjustsFontSizeToFitWidth = true
```

Comment:
Hi, @VladislavAntonyuk 
This fix makes the font size auto-adjust and smaller.
However, as the string gets longer, the font size does not decrease endlessly, but when 
it reaches the point where it can be wrapped, the font size returns to the specified size. 
What I wanted was for the font size not to decrease, but I haven't been able to find a way to do that.

Reply:
I also tried calling UILabel's SizeToFit method, but that didn't work as expected.

Reply:
I finally understand the essence of this issue. 

With zero padding, PaddedLabel word wrap works as intended. However, setting the padding of the PaddedLabel to a non-zero value will cause the message to be omitted when the length of the message approaches the display width.
Apparently Padding seems to be involved. I've seen similar behavior with MauiLabel in .NET MAUI. 

When calling the DrawText method, the InsetRect method is used to reduce the drawing rect by the amount of padding, but the actual text rect is not reduced by the amount of padding. There is a gap between the drawing rect and the text rect.

This deviation is the cause of the bug.
It can be solved by overriding the TextRectForBounds method and implementing it as follows.

    public override CGRect TextRectForBounds(CGRect bounds, nint numberOfLines)
    {
        var insets = new UIEdgeInsets(TopPadding, LeftPadding, BottomPadding, RightPadding);
        return base.TextRectForBounds(insets.InsetRect(bounds), numberOfLines);
    }

Below is the execution result.

[Snackbar]

Message:1234567890ABCD
![2023-07-06_01-51-55-午後](https://github.com/CommunityToolkit/Maui/assets/125236133/5dfcb861-134b-45db-8cfb-2e685a11d1c9)

Message:1234567890ABCDE
![2023-07-06_01-52-00-午後](https://github.com/CommunityToolkit/Maui/assets/125236133/91a19f65-83bf-476b-a40b-318a2ddd5ba0)

[Toast]

Message:1234567890ABCDEFG
![2023-07-06_01-52-31-午後](https://github.com/CommunityToolkit/Maui/assets/125236133/a2da7ea6-a1c6-4e38-9c5b-b1bc0b6b4376)

Message:1234567890ABCDEFGH
![2023-07-06_01-52-37-午後](https://github.com/CommunityToolkit/Maui/assets/125236133/007aa11c-2286-4ee8-aac1-a0db19702efd)

I was able to solve this problem without adjusting the font size.
Revert the last added change and add this fix.
Could you please review again? Thank you.